process/block/preprocess/transactionsV2.go

```diff
@@ -391,7 +391,7 @@ func (txs *transactions) verifyTransaction(
 		}
 
 		mbInfo.schedulingInfo.numScheduledBadTxs++
-		log.Trace("bad tx", "error", err.Error(), "hash", txHash)
+		log.Debug("bad tx", "error", err.Error(), "hash", txHash)
```

Comment:
Intended, important.

Reply:
no. It will fill up the txLogs with too many things. Revert.